Desktop/engine/enginesync.cpp

```diff
@@ -306,7 +306,7 @@ void EngineSync::start(int )
 	connect(timerBeat,		&QTimer::timeout, this, &EngineSync::heartbeatTempFiles,	Qt::QueuedConnection);
 
 	timerProcess->start(50);
```

Comment:
Why the timerBeat should be so often?

Reply:
It helps after wake up so files don't get deleted. But since the limit is 24 hours now it's not necessarily needed.
But on the other hand 50 milliseconds is still a lifetime so it doesn't matter for performance.